### DISCUSSION TITLE
Start program problem

### DISCUSSION THREAD

**Discussion by ccante29** [ArminJo/JK-BMSToPylontechCAN#35] [Q&A]
Hello 
Thanks for this great project. 

Unfortunately I have a problem... 
I use a JK B2A24S20P BMS with 16 280Ah cells. 
The information is accessible via Bluetooth on the BMS without any problems. 

So I have an Arduino uno with a screen. 
Once the Arduino is started without being connected to the BMS, it works well. 
The program starts, does not detect the CAN signal (this is normal because it is not connected). 
And then I have the message "Receive timeout". 
This is also normal. 
![Screenshot_20240712-142253](https://github.com/user-attachments/assets/09fb3732-754b-4b44-8006-8cb505276d21)

When I connect the Arduino to the BMS, I start the BMS, and then the Arduino. 
The problem is that the program does not start... 
The screen lights up but the characters remain black and nothing happens... 
![PXL_20240813_131853473](https://github.com/user-attachments/assets/2d5260f4-c354-4baa-aec6-3c6d5d478fa4)

If I start again with the BMS off it is the same. 
 If I redo with the ground disconnected it starts, but there is no signal received by the arduino.

I removed as many things as possible on the circuit to try to understand, but there I do not understand why it does not work...
![PXL_20240822_171012476](https://github.com/user-attachments/assets/12d08f63-8e48-469e-a375-014ab62de3ff)

To see if you have an idea of ​​the problem.
Thanks for everyone helps

**Comment by ArminJo** [comment]
Please post the output of the serial monitor.

**Comment by ArminJo** [comment]
On the photo it looks like you reversed RX and TX!
